INSTETC/INSTETC-IOC-01App/Db/INSTETC.db

```diff
@@ -17,6 +17,16 @@ record (longin, "$(P)CS:MOT:STOP:ALL")
 	field(MDEL, -1)
 }
 
+## String field for the status of motor movement
+record(bi, "$(P)CS:MOT:MOVING:STR") 
+{
+	field(DESC, "IOCs with motion to complete")
+    field(INP, "$(P)CS:MOT:MOVING CP MS")
+	field(ZNAM, "STATIONARY")
+    field(ONAM, "MOVING")
```

Comment:
I think it may be worth adding

field(ASG, "NOTRAPW")

it may not be necessary here, but no harm. It is to stop the "putlog" filling up with motor moves

Reply:
I thought as it was not being written externally it would not appear in the putlog anyway (it is just monitoring a local PV)?

Reply:
Putlog uses the trapwrite part of access security and that only applies to channel access, and normally channel access means external. This is processing initiated by a channel access monitor so i though it probably didn't apply, it depended if it treated the processing caused by a channel access monitor as ultimately being the same as a channel access write to the .PROC field (which should trigger a putlog entry). Maybe worth a test.